packages/checkbox/src/react/index.tsx

```diff
@@ -30,6 +30,9 @@ interface CheckboxProps
   value: string | number
 }
 
+/* eslint-disable-next-line */
+const noop = () => {}
```

Comment:
We typically want to avoid adding any hacks that force us to use eslint disables as it results in a code smell. I'll check this out further to see what else we can do to get this fixed and stay within out eslint boundaries.

Reply:
Sounds good! For additional context, I paired with @danethurber on this and I think we decided to go that route because checkbox and radio input elements look for `onChange` events; we're using an `onClick` event for the radio button and lacked any kind of `click` or `change` event on the checkbox. I believe we were seeing warnings logged in the tests, too. Because there's a refactor/overhaul of DS in the works, we decided to utilize a quick solution rather than refactor a bunch of code.

I'm happy with whatever you and Dane feel is best. Thanks!

Reply:
Awesome! Thanks for the context @dannytannertantrum . I went ahead and just extended the component to allow people to use an `onChange` if they wanted to (but not required) which resolved that issue. Thanks so much for the PR! 💪

Reply:
Great - thank you! I like that solution; nice work! 👏